src/Core/Application/RealTime/UseCase/FindHost/FindHost.php

```diff
@@ -81,13 +80,7 @@ public function __invoke(int $hostId, FindHostPresenterInterface $presenter): vo
         if ($this->contact->isAdmin()) {
             $host = $this->repository->findHostById($hostId);
             if ($host === null) {
-                $this->critical(
-                    "Host not found",
-                    [
-                        'id' => $hostId,
-                        'userId' => $this->contact->getId()
-                    ]
-                );
+                $this->handleHostNotFound($hostId, $presenter);
                 $presenter->setResponseStatus(new NotFoundResponse('Host'));
```

Comment:
You forgot to delete this line.

Reply:
Already patched here https://github.com/centreon/centreon/pull/10589/commits/5565a27029cf25fee3b353e442d98acbaebf020f

Reply:
✅  done